container/karaf/src/main/filtered-resources/etc/org.apache.karaf.features.cfg

```diff
@@ -91,6 +91,7 @@ featuresBoot = ( \
   opennms-poller-shell,\
   opennms-topology-runtime-graphml,\
   opennms-topology-runtime-asset,\
+  scv-jceks-impl,\
```

Comment:
This conflicts with other PR #4540
We need only single reference of the bean `JCEKSSecureCredentialsVault`. For metadata integration with SCV, we need to access this as spring bean, above PR adds this as spring bean and exposes it as OSGi reference on OpenNMS.

Adding this would expose multiple beans of SCV on OSGi registry.

Don't need this feature on OpenNMS once #4540 is merged.

Reply:
as discussed, I removed it. I will wait with this PR until https://github.com/OpenNMS/opennms/pull/4540 is merged.

Reply:
I moved spring bean related changes to #4547.  Once you merge develop, you should be good.

Reply:
done.